src/polyswarmd/websockets/serve.py

```diff
@@ -0,0 +1,35 @@
+import gevent
+
+from geventwebsocket import WebSocketServer, WebSocketApplication, Resource
+
+from .filter import (FilterManager)
+
+
+class EventServer(WebSocketApplication):
```

Comment:
Is it possible to deploy this via gunicorn? If so, should we setup a wsgi.py for websockets in addition to serve?

Reply:
I don't know, I've only run it independently (awhile ago). I am currently just keeping this in for future attempts to "isolate" websocket work.